### DISCUSSION TITLE
Audio Quality

### DISCUSSION THREAD

**Discussion by Mine181** [JunkFood02/Seal#376] [Ideas]
Would it be possible to add a quality selection for Audio-only downloads? or to be able to set a specific bitrate?

**Comment by JunkFood02** [comment]
Could you please more detailed info about this use case?

**Reply by Mine181** [reply]
As far as I know, YT-DLP has options to select the audio quality from 1 to 10 or to set a specific bitrate in kbps. 
Because Opus is very bitrate efficient you can lower the bitrate of audio files to something like 56 Kbps and the quality is very close if not indistinguishable from 128, at least for me.
Since I intend on downloading many videos for offline listening, I would like to limit their file size to something more reasonable.
Here are some quick UI mockups: 

![option](https://user-images.githubusercontent.com/83498665/204769915-f1b40516-85c1-454e-94f9-3bf9a7174a61.png)
![Menu 1a](https://user-images.githubusercontent.com/83498665/204769920-d5335129-bfeb-4d60-be8f-ccdd7ea73a86.png)
![Menu 1b](https://user-images.githubusercontent.com/83498665/204769923-38d496c3-97ed-4168-8428-5b8bbde6d361.png)
![Menu 2](https://user-images.githubusercontent.com/83498665/204770765-57880acf-e647-460c-8271-728111bc2653.png)

**Reply by JunkFood02** [reply]
Thanks for clarifying. This is indeed a practical usage, but I'm afraid it would not be added in the general mode due to possible abuse (primary users are likely to recode the audio files to get "higher" audio quality).

I would suggest to use a custom command to achieve this, if you mainly download the audio file from YouTube which already provides lower bitrate opus files, use `--no-mtime -f "ba[abr<60]" -x` is fine

**Reply by Mine181** [reply]
Thanks for the response!
